src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -45,6 +45,8 @@
 MAX_TAG_KEY_LENGTH = 200
 MAX_TAG_VALUE_LENGTH = 200
 
+ACCEPTED_METRIC_TYPES = {"s", "c", "d"}  # set, counter, distribution
```

Comment:
Is there a reason this isn't an Enum? This is just a nit, but it doesn't feel right to have arbitrary constants like this.

Reply:
No specific reason, but we're not doing anything with the metric type here besides validation so I just didn't really see a need for an internal representation